package.json

```diff
@@ -111,7 +111,9 @@
     "./lib/adapters/preferredAdapters.js": "./lib/adapters/preferredAdapters-browser.js",
     "./lib/deps/buffer.js": "./lib/deps/buffer-browser.js",
     "bluebird": "lie",
-    "fs": false
+    "fs": false,
+    "levelup": false,
```

Comment:
Ah yeah `levelup: false` will cause problems when trying to build the adapter plugins. I assume if we set them from the browserify command line in the [plugin build script](https://github.com/pouchdb/pouchdb/blob/7a7d0310afcc6cfca0509e36d75986f6ce62d313/bin/build-plugin.sh), it'll ignore this value in the package.json, yes?